admin_interface/migrations/0001_initial.py

```diff
@@ -225,4 +228,5 @@ class Migration(migrations.Migration):
                 "verbose_name_plural": "Themes",
             },
         ),
+        migrations.RunPython(create_default_theme),
```

Comment:
```suggestion
        migrations.RunPython(create_default_theme, reverse=migrations.RunPython.noop),
```
to allow reversing the migration

Reply:
Added it to the updated version